ambry-cloud/src/main/java/com/github/ambry/cloud/azure/CosmosDataAccessor.java

```diff
@@ -601,7 +601,7 @@ public long deprecateContainers(Set<CosmosContainerDeletionEntry> deprecatedCont
         }
       } catch (DocumentClientException dex) {
         if (dex.getStatusCode() == HttpConstants.StatusCodes.CONFLICT) {
-          logger.info("Container with accountid {} and containerid {} already exists. Skipping.",
+          logger.info("Container with accountid {} and containerid {} already deprecated. Skipping.",
```

Comment:
Could you please help me recall why we will see CONFLICT when container has been deprecated?